src/main.py

```diff
@@ -160,30 +240,37 @@ def compare_available_metrics():
 
     save_metric_definitions(current)
 
+
 def check_last_two_weeks_of_metrics():
-    threshold = PERCENTAGE_CHANGE_THRESHOLD
+    percentage_change_threshold = PERCENTAGE_CHANGE_THRESHOLD
+    cases_threshold = CASES_PER_100000_POPULATION_THRESHOLD
+
+    percentage_change_thresholds = {"percentage_change_threshold": percentage_change_threshold}
+    cases_thresholds = {"percentage_change_threshold": percentage_change_threshold, "metric_value_per_100000_threshold": cases_threshold}
 
     all_data = [
-        get_areas_above_threshold("nhsRegion", "newAdmissions", threshold, 'mean'),
-        get_areas_above_threshold("ltla", "newCasesBySpecimenDate", threshold, 'sum'),
-        get_areas_above_threshold("nhsRegion", "hospitalCases", threshold, 'mean')
+        get_areas_above_thresholds("nhsRegion", "newAdmissions", percentage_change_thresholds, 'mean'),
+        get_areas_above_thresholds("ltla", "newCasesBySpecimenDate", cases_thresholds, 'sum'),
+        get_areas_above_thresholds("nhsRegion", "hospitalCases", percentage_change_thresholds, 'mean')
     ]
 
     to_alert = [f"<p>{df.to_html()}</p>" for df in all_data if len(df) > 0]
 
     if len(to_alert) > 0:
         body = textwrap.dedent(f"""
-            <p>Some metrics have exceeded {threshold}% change week on week:</p>
+            <p>Some metrics have exceeded {percentage_change_threshold}% change week on week:</p>
```

Comment:
do we need a separate message for those that have exceeded the cases per 100K threshold?

Reply:
I thought about it, and decided against it. We'd be introducing metric level messages at that point, with a cumbersome sounding "newCasesBySpecimenDate have exceeded the thresholds 100% week on week and > 50 cases per 100,000 for the following areas:" 

If the email had a wider distribution list it might be worth being more explicit with the messages